backend/controllers/menu.js

```diff
@@ -6,11 +6,7 @@ const menuFactory = (prisonApi, logError) => {
       res.render('menu.njk', {
         title: 'Manage user accounts',
         homeUrl: config.app.notmEndpointUrl,
-        contentUrl: `${config.app.dpsEndpointUrl}/content`,
-        user: {
-          displayName: req.session.userDetails.name,
-          ...req.session.userRoles,
-        },
+        dpsUrl: `${config.app.dpsEndpointUrl}/content`,
```

Comment:
We push links into a nunjucks global as they tend to be used across multiple places